prow/plugins/config.go

```diff
@@ -661,6 +661,9 @@ type Welcome struct {
 	// MessageTemplate is the welcome message template to post on new-contributor PRs
 	// For the info struct see prow/plugins/welcome/welcome.go's PRInfo
 	MessageTemplate string `json:"message_template,omitempty"`
+	// Post welcome message in all cases, even if PR author is not an existing
+	// contributor or part of the organization
+	Everyone bool `json:"everyone,omitempty"`
```

Comment:
This is somewhat nitpicky, but what about `always_post`? `Everyone` is IMHO not really self-explanatory

Reply:
Not nitpicky at all, it does sound better
Fixed